figures/death_curves/scripts/deathcurves_plot.r

```diff
@@ -15,32 +15,34 @@ death_df <- read.csv(death_curve_path)
 head(death_df)
 
 # death percentage
-death_df$percentage_dead_cells <- death_df$Death.Cells/(death_df$Death.Cells + death_df$Live.Cells)	*100
-death_df$treatment <- paste0(death_df$Compound," ",death_df$Dose," ",death_df$unit, " ",death_df$time, " ", death_df$`unit.1`)
-head(death_df)
+# drop the NA values
+death_df <- death_df[!is.na(death_df$Dead_cells),]
+death_df$percentage_dead_cells <- death_df$Dead_cells/(death_df$Dead_cells + death_df$Live_cells)	*100
+death_df$treatment <- paste0(death_df$Compound," ",death_df$Dose," ",death_df$dose_unit, " ",death_df$time, " ", death_df$time_unit)
 # aggregate data by treatment
-death_df_agg <- death_df %>% group_by(treatment, Compound, unit) %>% summarise(mean_death = mean(percentage_dead_cells), sd_death = sd(percentage_dead_cells), time = mean(time), dose = mean(Dose))
+death_df_agg <- death_df %>% group_by(treatment, Compound, dose_unit) %>% summarise(mean_death = mean(percentage_dead_cells), sd_death = sd(percentage_dead_cells), time = mean(time), dose = mean(Dose))
 # make dose as factor
-death_df_agg$dose <- factor(death_df_agg$dose,levels = c("0","1","10"))
-death_df_agg$treatment <- paste0(death_df_agg$Compound," ",death_df_agg$dose, " ",death_df_agg$unit)
+death_df_agg$dose <- factor(death_df_agg$dose,levels = c("0","0.1","1","10"))
+death_df_agg$treatment <- paste0(death_df_agg$Compound," ",death_df_agg$dose, " ",death_df_agg$dose_unit)
 # factorize treatment
+unique(death_df_agg$treatment)
 death_df_agg$treatment <- factor(death_df_agg$treatment,levels = c(
     'Media 0 ',
-    'DMSO 0 ',
+    'DMSO 0.1 %',
     'LPS 1 ug/mL',
     'LPS 10 ug/mL',
-    'Thapsigargin 1 uM',
-    'Thapsigargin 10 uM'
+    'Thapsi 1 uM',
+    'Thapsi 10 uM'
```

Comment:
I don't see this name change in the plots, was this supposed to make a change for the figure or no?

Reply:
removed this portion